### PR TITLE
Delete temp VNC files that can cause bootloop

### PR DESCRIPTION
**Pull request by MDXDave**
See #100

### CODE REVIEW

File: docker/root/start.sh [hunk 1]
```diff
@@ -65,6 +65,9 @@ if [ "$GROUPPERMISSIONS" = "---" ]; then
   chmod g+rw $DEVICE
 fi
 
+# delete VNC temp files, that can cause bootloop (see #100)
```
Comment (Jo-Be-Co) [inline docker/root/start.sh:68]: Shouldn't this be put into the if-block where vnc is started? I would place it in the area where folders and links are created.
And imho for clearance the option `-f` should be put in front of the paths.

I was wondering why the folder `/tmp/.X11-unix/` is to be kept.

**Review by Jo-Be-Co — CHANGES_REQUESTED** [review]

File: docker/root/start.sh (hunk 1 above)
Comment (phdelodder) [reply docker/root/start.sh:68]: @MDXDave Please change it, to only remove the tmp files when VNC is going to be started.

**Comment by Jo-Be-Co** [comment]
One last addition to the names of the lock files.

If you use a different `DECONZ_VNC_PORT` the VNC will create a different display number and with it also different lock files. I.e. using Port 5909 will lead to the files `/tmp/.X9-lock` and `/tmp/.X11-unix/X9`.

So using wildcards is the right thing to do if you want to delete the files manually.

**Comment by phdelodder** [comment]
@MDXDave can you please implement the changes that @Jo-Be-Co recommends?

**Comment by Jo-Be-Co** [comment]
If need I could implement these in my own fork ... but I don't want to hijack this improvement.

**Comment by senilio** [comment]
> If needed I could implement these in my own fork ... but I don't want to hijack this improvement.

@Jo-Be-Co  Considering we're seeing reports from more and more users that would be helped by implementation, I vote for getting this change out in a separate PR. You can always credit @MDXDave for highlighting the issue and starting the discussion :)

**Comment by senilio** [comment]
Superseded by https://github.com/deconz-community/deconz-docker/pull/107

**Comment by patricstar1986** [comment]
Hello,

I still have the problem with VNC. 
Latest version is installed. 

Deleting the lockfiles X10-lock and /tmp/.X11-unix/X10 does not bring success.
I have set port 5910. 
Also changing to another one does not bring success. 
Does anyone else have a suggestion?
My version is 2.15.03, Synology Docker.

With version 2.14.01 everything works.

Thanks

**Comment by Jo-Be-Co** [comment]
My `2.15.03` is doing find on DSM7 ... could you show us your startup log?

You also could try to start with `DECONZ_UID=0` and `DECONZ_GID=0`.

**Comment by patricstar1986** [comment]
mit uid 0 zund gid 0 habe ich es schon versucht. 

Anbei das Log wenn ich VNC aktiviere. Ohne Vnc funktioniert es. 


[_deconzcommunity_deconz_logs.txt](https://github.com/deconz-community/deconz-docker/files/8657638/_deconzcommunity_deconz_logs.txt)

**Pull request closed without merging**